### DISCUSSION TITLE
Get content of a specific file

### DISCUSSION THREAD

**Discussion by michaelbilow** [SilentVoid13/Templater#1012] [Help]
I'm using Templater to manage my daily notes, and I want to get the content of yesterday's note so I can manipulate it to automatically create today's. I can get the content of the note I'm creating with `tp.file.content`, but what I want is to get the content of a specific note. What's the simplest way to do that?

**Comment by welpdx** [comment]
You can use a function like [tp.file.include()](https://silentvoid13.github.io/Templater/internal-functions/internal-modules/file-module.html#tpfileincludeinclude_link-string--tfile):

Example from [documentation](https://silentvoid13.github.io/Templater/internal-functions/internal-modules/file-module.html#examples):
```js
File Include: <% tp.file.include("[[YesterdaysNote]]") %>
```


~welp
